apps/masterbots.ai/components/routes/profile/user-thread-list.tsx

```
@@ -0,0 +1,92 @@
+'use client'
+import { getBrowseThreads, getThreads } from "@/services/hasura";
+import UserThreadPanel from "../thread/user-thread-panel";
+import { useProfile } from "@/lib/hooks/use-profile";
+import { Thread, User } from "mb-genql";
```

Biome

[error] All these imports are only used as types.

Importing the types with import type ensures that they are removed by the compilers and avoids loading unnecessary modules.
Safe fix: Use import type.


(lint/style/useImportType)